### PR TITLE
[ENH] Add class for the AlphaIMS implant

### PR DESCRIPTION
**Pull request by oliver-contier**
... and added respective isinstance checks and conditional sit.resize (in stimuli.py).

The implant parameters are from the following paper:

`
@article{Stingl2015,
author = {Stingl, Katarina and Bartz-Schmidt, Karl Ulrich and Besch, Dorothea and Chee, Caroline K. and Cottriall, Charles L. and Gekeler, Florian and Groppe, Markus and Jackson, Timothy L. and MacLaren, Robert E. and Koitschev, Assen and Kusnyerik, Akos and Neffendorf, James and Nemeth, Janos and Naeem, Mohamed Adheem Naser and Peters, Tobias and Ramsden, James D. and Sachs, Helmut and Simpson, Andrew and Singh, Mandeep S. and Wilhelm, Barbara and Wong, David and Zrenner, Eberhart},
doi = {10.1016/j.visres.2015.03.001},
journal = {Vision Research},
pages = {149--160},
publisher = {Elsevier Ltd},
title = {{Subretinal Visual Implant Alpha IMS - Clinical trial interim report}},
url = {http://dx.doi.org/10.1016/j.visres.2015.03.001},
volume = {111},
year = {2015}
}
`

### CODE REVIEW

**Comment by mbeyeler** [comment]
Hi Oliver,

Thanks for your contribution - this is great!

Would you mind also adding a test case in `pulse2percept/tests/test_implants.py` similar to `test_ArgusII`?
It would be good to make sure that electrode sizes, numbers, names, etc. are set correctly. So you would instantiate your object, then compare parameter values to expected values (using e.g. `npt.assert_equal` or `npt.assert_almost_equal`). See this function here for a template:
https://github.com/uwescience/pulse2percept/blob/master/pulse2percept/tests/test_implants.py#L241

Thanks,
Michael

File: pulse2percept/stimuli.py [hunk 1]
```diff
@@ -295,7 +295,8 @@ def image2pulsetrain(img, implant, coding='amplitude', valrange=[0, 50],
 
     isargus1 = isinstance(implant, implants.ArgusI)
     isargus2 = isinstance(implant, implants.ArgusII)
-    if not isargus1 and not isargus2:
+    isalphaims = isinstance(implant, implants.AlphaIMS)
+    if not isargus1 and not isargus2 and no isalphaims:
```
Comment (mbeyeler) [inline pulse2percept/stimuli.py]: There's a typo in `no isalphaims`

Comment (mbeyeler) [reply pulse2percept/stimuli.py]: Found a way to fix it myself. :)

**Comment by arokem** [comment]
Looks like we still need a test-case for this, right?

**Comment by mbeyeler** [comment]
@arokem: Yep. I was trying to add it myself, but it looks I can only edit files that have been changed as part of the PR. The test file isn't one of them.

**Comment by arokem** [comment]
Hmm. If @oliver-contier doesn't do it in the next few weeks, you can also fetch his branch down to your machine, do the edits on a branch derived from this branch, and then push it up as a new PR.

**Pull request merged**